src/test/java/picard/vcf/VcfTestUtils.java

```diff
@@ -12,6 +12,31 @@
 import java.util.EnumSet;
 
 public class VcfTestUtils {
+
+    /**
+     * This method creates a temporary VCF file and it's appropriately named index file, and will delete them on exit.
+     * @param prefix - The prefix string to be used in generating the file's name; must be at least three characters long
+     * @param suffix - The suffix string to be used in generating the file's name; may be null, in which case the suffix ".tmp" will be used
+     * @return A File object referencing the newly created temporary VCF file
+     * @throws IOException - if a file could not be created.
+     */
+    public static File createTemporaryIndexedVcfFile(final String prefix, final String suffix) throws IOException {
```

Comment:
the two functions createTemporaryIndexedVcfFile and createIndexedVcf do different things, but have misleadingly similar names...could you make sure that the names make it more clear what the differences are, or alternatively, rebase them into one function with an additional argument?